leaderboard/trueskill_ranking/trueskill_ranking.py

```diff
@@ -0,0 +1,37 @@
+import random
+
+import trueskill
+from tqdm import tqdm
+
+
+def compute_leaderboard(agents_name, evaluate_agents, sigma_threshold=1.):
+    # Allows sigma to always converge
+    trueskill.setup(tau=0.)
+    # Create a rating object for each agents
+    ratings = {agent_name: trueskill.Rating() for agent_name in agents_name}
+
+    pbar = tqdm()
+    pbar.set_description("Computing TrueSkill leaderboard")
+    while True:
+        # Find agent with bigger sigma
+        agent_a, sigma_max = None, -1
+        for agent_name, rating in ratings.items():
+            if rating.sigma > sigma_max:
+                agent_a, sigma_max = agent_name, rating.sigma
+        pbar.set_postfix({"sigma_max": sigma_max})
+        # Stop if all sigma are smaller than the sigma_threshold
+        if sigma_max < sigma_threshold:
+            break
+        # Find another agent at ramdom
+        agent_b = random.choice([agent_name for agent_name in ratings if agent_name != agent_a])
+        # Evaluate agents together and update their ratings
+        result = evaluate_agents(agent_a, agent_b)
+        if result == -1:
```

Comment:
why do we swap agetns if agent a loses?

Reply:
This was to not write two times this line:
`ratings[agent_a], ratings[agent_b] = trueskill.rate_1vs1(ratings[agent_a], ratings[agent_b], drawn=drawn)`
Maybe it isn't easy to read indeed, it is kind of Leetcode habit. I can change this.

Reply:
Ah okay it's probably something about trueskill I didn't understand. it's a way of swapping the rating order?

It's probably okay for our purposes and I'm happy to merge, but you're obviously welcome to change it :-)